splink/spark/spark_linker.py

```diff
@@ -167,6 +177,23 @@ def __init__(
             input_table_aliases=homogenised_aliases,
         )
 
+        # check to see if running in databricks and use delta lake tables as break lineage method.
+        self.in_databricks = "DATABRICKS_RUNTIME_VERSION" in os.environ
+        if self.in_databricks:
```

Comment:
I wonder whether we want this to run only if the user hasn't explicitly set a break lineage method?  

As it stands, doesn't this mean that, in databricks, `delta_lake_table` will always be used, even when the user has explicitly asked for a different `break_lineage_method`?

I think this is slightly more fiddly than it looks to achieve because `break_lineage_method` defaults to `"parquet"`, so it's hard to distinguish an explicit 'parquet' value from an implicit (default) parquet value.  I guess one option would be to change `break_lineage_method` to default to None, and if None is encountered, set `self.break_lineage_method` to parquet on line 121

Reply:
Thinking about this a bit more - should we also change the default save format to delta if we believe it offers better performance?  

Or at the performance improvements more pronounced in databricks?

Are the performance improvement restricted to iterative cases where we're 'rewriting' a table multiple times that's changing only a subset of the data each iteration, or is this a better way to break long lineages even if the table is written just once?

Reply:
I did wonder the same - I like the approach about updating if none. I was worried about breaking any existing workflows by changing the default, but setting to parquet if none sounds reasonable.

Reply:
Great.  Yeah - good to be cautious! - I think so long as we set to `parquet` if it's None, then we should be safe and not break any existing workflows, so I'm happy to go ahead with that

Reply:
re the switch to Delta as default - I'm not so sure about this. Delta offers benefits over parquet, but may not be immediately replaceable - it depends if people are using the intermediate steps in their workflows. 

I could see a benefit for any iterative workloads with Delta Caching, but that's a Databricks specific feature. This would be one worth discussing in more detail, and also doing some testing around to see if the difference is meaningful